Source/WebCore/platform/graphics/GraphicsStyle.h

```diff
@@ -73,6 +73,11 @@ inline bool operator==(const GraphicsColorMatrix& a, const GraphicsColorMatrix&
     return a.values == b.values;
 }
 
+inline bool operator!=(const GraphicsColorMatrix& a, const GraphicsColorMatrix& b)
+{
+    return !(a.values == b.values);
```

Comment:
!(a == b)

Reply:
Contains the equality logic just in the == operator, and takes the != operator as a direct negation of that.